keyboards/handwired/hacked_motospeed/info.json

```diff
@@ -0,0 +1,93 @@
+[
```

Comment:
This file isn't valid for QMK Configurator support. This JSON is Keyboard Layout Editor format, but QMK Configurator uses a different format.

The QMK Docs detail how to do this, but because it depends on code that @drashna has requested you change, do that first, and then we'll come back to this.

Reply:
I didn't fully read the documentation before, my bad - it is fixed now.